### PR TITLE
create responsive css file

### PR DESCRIPTION
**Pull request by tugsem**
In this branch I :

-created new css file 
-added media queries with max-width: 768px;
-changed layout 
-removed hamburger button and add navigation links to header

### CODE REVIEW

File: css/responsive.css [hunk 1]
```diff
@@ -0,0 +1,135 @@
+@media all and (min-width: 768px) {
```
Comment (TedLivist) [inline css/responsive.css:1]: There seems to be an issue with your desktop version of the CSS. Right after moving to the desktop version and up to a certain width, your portfolio has some misalignments and leaves a lot of spaces to the right.
- [ ] The **navbar** and **the banner** has some space to the right. Both of them should occupy the full width at all times. Kindly fix this.

- [ ] The listed works only align to the left, but they should be at the center of the page (leaving equal margins to the left and right at all times). Kindly fix this.

- [ ] The footer section also has some space to the right. It should cover the full width of the screen at all times.

  Navbar, banner, and listed works | Footer section
  :---:|:---:
  tugsem-html | tugsem-footer

File: index.html [hunk 2]
```diff
@@ -27,94 +33,102 @@ <h4 class="connect">LET'S CONNECT</h4>
...
+                </p>
+                <ul class="languages">
+                    <li class="lang">html</li>
+                    <li class="lang">css</li>
+                    <li class="lang">javascript</li>
+                </ul>
+                <button type="button" class="detail-btn">See Project</button>    
+            </div>
+    
         </section>
```
Comment (TedLivist) [inline index.html:127]: - [ ] Kindly ensure you follow the desktop design as we have in the template. In the Figma template design, we have alternating image positions whereas, in your implementation, all the images are aligned to the left. Kindly fix this.

  Your Implementation | Figma Design
  :---:|:---:
  tugsem-images | tugsem-images-figma

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @tugsem,

Good job so far!
✔️ Good documentation
✔️ Nice design
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -27,94 +33,102 @@ <h4 class="connect">LET'S CONNECT</h4>
...
+                <p class="description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="languages">
+                    <li class="lang">html</li>
+                    <li class="lang">css</li>
+                    <li class="lang">javascript</li>
+                </ul>
+                <button type="button" class="detail-btn">See Project</button>    
+            </div>
```
Comment (Omar-Muhamad) [inline index.html:56]: - [ ] For all work cards please add the right width value and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image |  image |

**Review by Omar-Muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @tugsem,

Nice job making the required changes. Unfortunately, you missed some of them and there are other problems but you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] For the contact form please add the right width value because your design width is smaller and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image  | image  |

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by prolajumokeoni — APPROVED** [review]
Hi @tugsem, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**